src/main/java/org/commcare/resources/model/ResourceTable.java

```diff
@@ -872,7 +874,28 @@ public void clear(CommCarePlatform platform) {
         if (count > 0) {
             Logger.log(LogTypes.TYPE_RESOURCES, "Cleaned up " + count + " records from table");
         }
+        storage.removeAll();
+    }
 
+    /**
+     * Destroy this table, and remove any files installed by it.
+     */
+    public void clearAll(CommCarePlatform platform) {
```

Comment:
This is so close to the one above that I would merge them even if it makes the resulting method more complex